src/rp2040/rp2040_link.lds.S

```diff
@@ -20,20 +20,21 @@ SECTIONS
     .text : {
         . = ALIGN(4);
         KEEP(*(.boot2))
-        _text_vectortable_start = .;
-        KEEP(*(.vector_table))
-        _text_vectortable_end = .;
-        *(.text .text.*)
-        *(.rodata .rodata*)
+        KEEP(*(.vector_table_flash))
+        KEEP(*(.reset_handler_flash))
+        KEEP(*(.reset_handler_flash.*))
     } > rom
 
     . = ALIGN(4);
     _data_flash = .;
 
     .data : AT (_data_flash)
     {
-        . = ALIGN(4);
+        . = ALIGN(128);
```

Comment:
Actually ALIGN do not do anything, cause we are at the start of the ram anyway :)